src/components/_app/Web3ConnectionManager/hooks/useWeb3ConnectionManager.ts

```diff
@@ -0,0 +1,141 @@
+import { CHAIN_CONFIG, Chains } from "chains"
+import useToast from "hooks/useToast"
+import { atom, useAtom } from "jotai"
+import { useRouter } from "next/router"
+import { useEffect } from "react"
+import { PlatformName } from "types"
+import { useAccount, useSwitchNetwork } from "wagmi"
+
+const delegateConnectionAtom = atom(false)
+const safeContextAtom = atom(false)
+const walletSelectorModalAtom = atom(false)
+const accountModalAtom = atom(false)
+const platformMergeAlertAtom = atom(false)
+const accountMergeAddressAtom = atom("")
+const accountMergePlatformNameAtom = atom("" as PlatformName)
```

Comment:
Would it make sense to export these atoms instead of returning them from the hook (I mean the ones where only the value and the setter are returned)

Reply:
Note: we talked about it irl, we'll leave it as is for now